src/ray/core_worker/task_interface.h

```diff
@@ -75,6 +82,10 @@ class ActorHandle {
   /// It's used to make sure ids of actor handles are unique.
   int64_t NumForks() const;
 
+  /// Whether direct call is used. If this is true, then the tasks
+  /// are submitted directly to the actor without going through raylet.
+  bool IsDirectCall() const;
```

Comment:
```suggestion
  bool IsDirectCallActor() const;
```

would read more nicely